doc/content/integrations/webhooks/path-variables.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Webhook Path Variables"
+description: ""
+weight: -1
+---
+
+Webhook path variables allow you to substitute device and application specific variables in webhook paths. This section provides instructions for using webhook path variables.
+
+<!--more-->
+
+Webhook path variables allow you to use the following variables in webhook paths:
+
+- `appID`
+- `appEUI`
+- `joinEUI`
+- `devID`
+- `devEUI`
+- `devAddr`
+
+Path variables can be inserted in the **Base URL** webhook field or the **Path** field for a particular type of message.
```

Comment:
I'd add a reference to the [RFC](https://tools.ietf.org/html/rfc6570), since there are a lot of ways you can actually replace the variables in the URL itself, and we do support them.